boringtun/src/noise/timers.rs

```diff
@@ -322,14 +305,12 @@ impl Tunn {
         TunnResult::Done
     }
 
-    pub fn time_since_last_handshake(&self) -> Option<Duration> {
-        let current_session = self.current.load(Ordering::Acquire);
-        if self.sessions[current_session % super::N_SESSIONS]
-            .read()
-            .is_some()
-        {
+    pub(super) fn time_since_last_handshake(&self) -> Option<Duration> {
+        tracing::error!(?self.sessions, "Sessions");
```

Comment:
Should this still be here as `tracing::error` or was it just added for debugging?

Reply:
Looks like it was removed during my review!

Reply:
Yeah, it was added for debugging the underflow issue.